src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -171,7 +171,12 @@ def put(self, request: Request, project, rule) -> Response:
 
             trigger_sentry_app_action_creators_for_issues(actions=kwargs.get("actions"))
 
-            if rule.data["conditions"] != kwargs["conditions"]:
+            condition_data = rule.data["conditions"].copy()
+            for condition in condition_data:
+                if condition.get("name"):
+                    del condition["name"]
+
+            if condition_data != kwargs["conditions"]:
                 metrics.incr("sentry.issue_alert.conditions.edited", sample_rate=1.0)
```

Comment:
Is there a way to search for charts in Datadog using a specific metric? I had to add this because we track this.

Reply:
I git blamed this to Andrew's intern project:  https://github.com/getsentry/sentry/pull/41051 but I can't find anything in use in Datadog (searching for incompatible rules), and I can't even find a spec for it that I'd hope would link any dashboard. I might just remove it.